### ISSUE TITLE
Scan Fails, Servers up and working

### ISSUE THREAD

**Issue by Contra28** [dgoguerra/pokego-scan#7]
I used the debugger to check what was going on here and it shows successes on the scans then moves from in progress to failed t. Is this still due laggy servers?

 DEBUG=pokego-scan pokego-scan -- 45.423312,-75.698452
  pokego-scan https://pokevision.com/map/scan/45.423312/-75.698452 +0ms
  pokego-scan { statusCode: 200, body: { status: 'success', jobId: '6d1d6a4944dbe4a8e4886f3bd7b19b10' } } +154ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +17ms
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +78ms
  pokego-scan job in progress, retry in 1500ms (attempt 2 of 10) +0ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +2s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +66ms
  pokego-scan job in progress, retry in 3000ms (attempt 3 of 10) +1ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +3s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +63ms
  pokego-scan job in progress, retry in 4500ms (attempt 4 of 10) +1ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +5s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +76ms
  pokego-scan job in progress, retry in 6000ms (attempt 5 of 10) +0ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +6s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +1s
  pokego-scan job in progress, retry in 7500ms (attempt 6 of 10) +0ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +8s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'in_progress' } } +74ms
  pokego-scan job in progress, retry in 9000ms (attempt 7 of 10) +0ms
  pokego-scan https://pokevision.com/map/data/45.423312/-75.698452/6d1d6a4944dbe4a8e4886f3bd7b19b10 +9s
  pokego-scan { statusCode: 200, body: { status: 'success', jobStatus: 'failure' } } +76ms
error: Unable to scan for pokemon. If this continues to fail then the Pokemon servers are currently unstable or offline.

**Comment by zosiu** [comment]
@Contra28 I just run the same thing you did, and I got results back, so yeah, most probably server lag/issues.

**Comment by Contra28** [comment]
Sounds good to me, will try later

**Comment by dgoguerra** [comment]
Thats right, its an error from the server, nothing to do there other than rescanning.
